pkg/sources/adapter/slacksource/slack.go

```diff
@@ -122,7 +124,7 @@ func (h *slackEventAPIHandler) handleAll(w http.ResponseWriter, r *http.Request)
 	// There are only 2 documented types to be received from the Events API
 	// - `event_callback`, See: https://api.slack.com/events-api#receiving_events
 	// - `event_callback`, See: https://api.slack.com/events-api#subscriptions
-	switch event.Type {
+	switch eventType := sanitizeUserInput(event.Type); eventType {
```

Comment:
This event type is eventually logged **inline**, like `"Content not supported: \"Foo\""`.
Considering the nature of the input (a single word), I don't think it is worth logging it as a structured field, and therefore decided to sanitize the input itself, instead of just the logged value.